src/main/java/net/minecraftforge/registries/IForgeRegistryEntry.java

```diff
@@ -29,7 +29,7 @@
 {
     /**
      * Sets a unique name for this Item. This should be used for uniquely identify the instance of the Item.
-     * This is the valid replacement for the atrocious 'getUnlocalizedName().substring(6)' stuff that everyone does.
+     * This is the valid replacement for the atrocious 'getTranslationKey().substring(6)' stuff that everyone does.
      * Unlocalized names have NOTHING to do with unique identifiers. As demonstrated by vanilla blocks and items.
```

Comment:
Either leave this as-is, or change "Unlocalized names..." to "Translation keys..." as well.